src/chart/FtpHistoryChart.svelte

```diff
@@ -0,0 +1,80 @@
+<script>
+    import { onMount } from "svelte";
+    import Chart from "chart.js";
+    import Card from "../components/Card.svelte";
+    import * as ChartAnnotation from "chartjs-plugin-annotation";
+    import { ftpAverageOutputs } from "../store/store.js";
+    import moment from "moment";
+
+    const getPlotPoints = (data) => {
+        return data.map((ride) => {
+            // TODO (tmack) find why there are duplicates?
+            console.log(ride);
```

Comment:
Let's get rid of this console log before merge, too.

Reply:
ah yes, that is resolved I think so I'll verify and remove it

Reply:
removed this as well